fh-amqp-js.js

```diff
@@ -5,7 +5,7 @@ var util = require('util');
 // Default config values
 var rc = require('rc')('fh-amqp-js', {
   maxReconnectAttempts: 10,
-  ebabled: true
+  enabled: true
```

Comment:
unless it is doing something with bable I presume this is write

Reply:
I wonder if fixing this will have any consequences though. I would be inclined to leave it as is unless there is a reason to change it?